mcp/src/test/java/io/airlift/mcp/TestMcp.java

```diff
@@ -107,7 +107,7 @@ public TestMcp()
 
         httpClient = injector.getInstance(Key.get(HttpClient.class, ForTest.class));
         baseUri = injector.getInstance(HttpServerInfo.class).getHttpUri().resolve("/mcp");
-        objectMapper = injector.getInstance(ObjectMapper.class);
+        jsonMapper = injector.getInstance(JsonMapper.class);
```

Comment:
Why is this needed? I manually changed this back to `ObjectMapper` and it still compiles.